### PR TITLE
Optimized ObservableEmitter serialization

### PR DESCRIPTION
**Pull request by arkivanov**

### CODE REVIEW

File: detekt.yml [hunk 1]
```diff
@@ -209,7 +209,7 @@ formatting:
     continuationIndentSize: 4
   MaximumLineLength:
     active: true
-    maxLineLength: 120
+    maxLineLength: 140
```
[the remark this answers is not in the document]
Comment (arkivanov) [reply detekt.yml]: Hmm, like in our Badoo code? We have 140 limit.

[the remark this answers is not in the document]
Comment (arkivanov) [reply detekt.yml]: In projects settings we have 140 defined, in `gradle.yml` even 160.
image

[the remark this answers is not in the document]
Comment (CherryPerry) [reply detekt.yml]: Maybe let's keep it until https://github.com/badoo/Reaktive/issues/241 is done?
AS/IDEA has 120 by default.

[the remark this answers is not in the document]
Comment (arkivanov) [reply detekt.yml]: If you have 120 set in the settings then you will never hit threshold and detent will be fine. But if you have 140  (like me) then detekt fails on long (actually ok) lines. What's the point?

**Pull request merged**